golem-test-framework/src/dsl/mod.rs

```diff
@@ -1352,11 +1354,11 @@ impl<T: TestDependencies + Send + Sync> TestDsl for T {
     }
 
     async fn check_oplog_is_queryable(&self, worker_id: &WorkerId) -> crate::Result<()> {
-        let _oplog = TestDsl::get_oplog(self, worker_id, OplogIndex::INITIAL).await?;
+        let oplog = TestDsl::get_oplog(self, worker_id, OplogIndex::INITIAL).await?;
 
-        // for (idx, entry) in oplog.iter().enumerate() {
-        //     debug!("#{}: {entry:#?}", idx + 1);
-        // }
+        for (idx, entry) in oplog.iter().enumerate() {
```

Comment:
do you want to keep this?

Reply:
Yes because now it has a nice output that does not take infinite space :)

Reply:
Would it worth putting the loop inside enabled(debug)?

Reply:
Why? :) 
This is a very specific check executed in the end of executor tests only, not affecting anything else

Reply:
sorrry, missed that this is test code